components/01-atoms/divider/_divider.scss

```diff
@@ -0,0 +1,36 @@
+$widths: 100%, 75%, 50%, 25%;
```

Comment:
Maybe you did this in a recent PR that I havent got to yet but it seems that these values would be better defined as tokens. Widths could also relate to grids, thickness may relate to borders, positions could be used for alignments in other components. You could probably re-assign those token values in this file to be something that is divider name-spaced in this file that you then loop through.

Ignore comment if I just haven't got to that change yet. Otherwise, I dig the logic in this file and re-assigning left/right to flex specific values makes sense.

Reply:
I hadn't done this yet, and I was able to figure it out. Should be there in the latest commits.

If you're curious, here's the associated tokens repo PR: https://github.com/yalesites-org/tokens/pull/12